exercises/01-getting-started.md

```diff
@@ -1,63 +1,77 @@
+>
+> We recommend to create your project as a private repository due to licensing constraints. In [exercise 2](02-generate-odata-clients.md) we will generate OData clients for SAP S/4HANA and SAP SuccessFactors, that is subject to SAP intellectual property and should not be made available as open source. As long as you don't publish these changes, feel free to create a public repository as well.
 
 ## Clone your repository
+
 On your laptop open a command line and enter the following command. Don't forget to replace your username (and repository name should you have chosen a different one):
+
 ```sh
 git clone https://github.com/<your-github-username>/cloud-sdk-team-calendar.git
 ```
 
 ## Deploy the SAP S/4HANA and SAP SuccessFactors mock server (Optional)
 
->Note: If you have access to real SAP S/4HANA Cloud and SAP SuccessFactors (SFSF) Cloud systems, you may be able to skip this step.
+> Note: If you have access to real SAP S/4HANA Cloud and SAP SuccessFactors (SFSF) Cloud systems, you may be able to skip this step.
 
-Follow these [instructions](https://github.com/SAP/cloud-s4-sdk-book/tree/mock-server#how-to-run-the-server) on how to setup your SAP S/4HANA and SFSF mock server and deploy it on SAP Business Technology Platform (SAP BTP), Cloud Foundry environment.
+Follow these [instructions](https://github.com/SAP/cloud-s4-sdk-book/tree/mock-server#how-to-run-the-server) on how to set up your SAP S/4HANA and SAP SuccessFactors mock server and deploy it on the SAP Business Technology Platform (SAP BTP), Cloud Foundry environment. 
 
-## Setup the destinations environment variable
+**Please checkout the branch [`mock-server-update-data`](https://github.com/SAP/cloud-s4-sdk-book/tree/mock-server-update-data) for this tutorial instead of the branch [`mock-server`](https://github.com/SAP/cloud-s4-sdk-book/tree/mock-server) as it is described in the instructions.** The branch has the latest date data as opposed to the branch `mock-server`.
 
-Replace the URL placeholders for the `S4HANA` and `SFSF` destinations in the [`.env`](../.env) file with the URL(s) of your mock server or your SAP S/4HANA and SAP SuccessFactors systems. If you are using the mock server, the URLs for both destinations are the same, because the mock server acts as both, a mocked SAP S4/HANA system and a mocked SAP SuccessFactors system.
+## Setup the destinations environment variable
 
+Replace the URL placeholders for the `S4HANA` and `SFSF` destinations in the [`.env`](../.env) file with the URL(s) of your mock server or your SAP S/4HANA and SAP SuccessFactors systems. If you are using the mock server, the URLs for both destinations are the same, because the mock server acts as both, a mocked SAP S4/HANA system and a mocked SAP SuccessFactors system. The mock server has no authentication to access it by default. Please set the authentication type `NoAuthentication` when you use the mock server.
```

Comment:
[q] @marikaner should we add a  sentence what to do if you have a real system? Like set env for local test with basic?